consultation_analyser/consultations/dummy_data.py

```diff
@@ -73,6 +73,18 @@ def create_dummy_data(responses=10, include_themes=True, number_questions=10, **
                 ans.themes.clear()
                 answers.append(ans)
 
+        # cause the last question to have answers with two responses to the multiple choice
+        # so that we always have a path to test this through the frontend
+        q = questions[-1]
```

Comment:
Is this right?

I'm getting some repeated answers (in the same consultation response) e.g. 
```
        {
          "multiple_choice": [
            { "options": ["Yes", "No"], "question_text": "Do you agree?" }
          ],
          "free_text": "Their packaging, often colorful and eye-catching, adds to the appeal.",
          "question_id": "83cdc1bf-62e5-417e-a53c-12c2567aa513"
        },
        {
          "multiple_choice": [
            { "options": ["No", "Yes"], "question_text": "Do you agree?" }
          ],
          "free_text": "",
          "question_id": "83cdc1bf-62e5-417e-a53c-12c2567aa513"
        }
      ]
 ```

Reply:
Maybe have a look if you generate some dummy data?

Reply:
Discussed: dummy data sometimes creates 2x answers for same resp/question combination